public/js/show-more.js

```diff
@@ -0,0 +1,43 @@
+window.addEventListener('load', init)
+
+function init() {
+
+    let btn = document.querySelector("#show-more")
+    let loader = document.querySelector(".loader")
+    let arrow = document.querySelector(".arrow-down")
+    let span = document.querySelector(".content-span")
+
+    btn.addEventListener("click", function (e) {
+        btn.style.visibility = "hidden";
+        loader.style.visibility = "visible"
+
+        let tricks = document.querySelectorAll(".js-card");
+        let lastTrick = tricks[tricks.length - 1].getAttribute("id");
+        const Params = new URLSearchParams();
+
+        Params.append("id", lastTrick);
+        const Url = new URL(window.location.href)
+
+        fetch(Url.pathname + "?" + Params.toString() + "&load=1", {
+            headers: {
+                "X-Requested-With": "XMLHttpRequest"
+            }
+        }).then(response => response.json()
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/VinzOo93/P6SnowTricks/pullRequest?prid=8761575)
```suggestion
        }).then((response) => response.json()
```